### PR TITLE
feat(dashboard): workflow list empty states

### PR DESCRIPTION
**Pull request by ChmaraX**
### What changed? Why was the change needed?

Workflow list empty states and copywriting according to Figma designs.

image

image

### CODE REVIEW

File: apps/dashboard/src/components/icons/version-control-dev.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import type { HTMLAttributes } from 'react';
+
+type VersionControlDevProps = HTMLAttributes<HTMLOrSVGElement>;
+export const VersionControlDev = (props: VersionControlDevProps) => {
```
Comment (LetItRock) [inline apps/dashboard/src/components/icons/version-control-dev.tsx:4]: please run svgs through the: https://jakearchibald.github.io/svgomg/
and update the attributes from dashes to camel case for example `stroke-opacity` to `strokeOpacity`

File: apps/dashboard/src/components/icons/version-control-prod.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import type { HTMLAttributes } from 'react';
+
+type VersionControlProdProps = HTMLAttributes<HTMLOrSVGElement>;
+export const VersionControlProd = (props: VersionControlProdProps) => {
```
Comment (LetItRock) [inline apps/dashboard/src/components/icons/version-control-prod.tsx:4]: same

File: apps/dashboard/src/components/workflow-list-empty.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    <div className="flex items-center justify-center gap-6">
+      <Link
+        to={'https://docs.novu.co/concepts/workflows'}
+        target="_blank"
+        className={buttonVariants({ variant: 'link', className: 'text-foreground-600 gap-1' })}
+      >
+        <RiBookMarkedLine className="size-4" />
+        View docs
+      </Link>
+      <CreateWorkflowButton asChild>
```
Comment (LetItRock) [inline apps/dashboard/src/components/workflow-list-empty.tsx]: there is no need to wrap in the `CreateWorkflowButton` I guess

File: apps/dashboard/src/components/icons/version-control-dev.tsx (hunk 1 above)
Comment (ChmaraX) [reply apps/dashboard/src/components/icons/version-control-dev.tsx:4]: Very cool, didn't know about that

**Review by LetItRock — APPROVED** [review]

**Pull request merged**